MAUI/AIAssistView/working-with-aiassistview.md

```diff
@@ -0,0 +1,236 @@
+---
+layout: post
+title: Working with AI AssistView in .NET MAUI | Syncfusion
+description: Learn here all about Working with AI AssistView support in Syncfusion .NET MAUI  AI AssistView control and more.
+platform: MAUI
+control: SfAIAssistView
+documentation: ug
+---
+
+# Working with AI AssistView in .NET MAUI AI AssistView (SfAIAssistView)
+
+## Stop Responding in .NET MAUI AI AssistView (SfAIAssistView)
+
+The [SfAIAssistView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.AIAssistView.SfAIAssistView.html) control provides `Stop Responding` feature that allows you to cancel an ongoing AI response by clicking the Stop Responding button. This feature ensures that users can stop if a response is no longer needed.
+By default, the Stop Responding button is displayed, to disable this set the `EnableStopResponding` property to `false`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+    <syncfusion:SfAIAssistView x:Name="sfAIAssistView"
+                               EnableStopResponding="False"/>  
+
+{% endhighlight %} 
+{% highlight c# hl_lines="6" %}
+
+    SfAIAssistView sfAIAssistView; 
+    public MainPage() 
+    { 
+        InitializeComponent(); 
+        this.sfAIAssistView = new SfAIAssistView();
+        this.sfAIAssistView.EnableStopResponding = false;
+        this.Content = sfAIAssistView; 
+     } 
+
+{% endhighlight %}
+{% endtabs %}
+
+### Stop Responding Event and Command
+
+The `SfAIAssistView` control includes a built-in event called `StopResponding` and a command named `StopRespondingCommand`. These are triggered when the Stop Responding button is clicked.
```

Comment:
The `SfAIAssistView` control includes a built-in event called `StopResponding` and a command named `StopRespondingCommand`. These are triggered when the `Stop Responding` button is clicked.

Reply:
Highlighted the line